src/extension.ts

```diff
@@ -732,17 +731,105 @@ export async function activate(
       )
     );
   }
-  // === Commands only for testing ===
+  // === End of commands only for testing ===
 
   const parseCompilerArgsScript: string = util.parseCompilerArgsScriptFile();
 
   // The extension VSIX stripped the executable bit, so we need to set it.
   // 0x755 means rwxr-xr-x (read and execute for everyone, write for owner).
   await fs.chmod(parseCompilerArgsScript, 0o755);
 
-  if (configuration.getConfigureOnOpen() && extension.getFullFeatureSet()) {
-    // Always clean configure on open
-    await make.cleanConfigure(make.TriggeredBy.cleanConfigureOnOpen);
+  // TODO: Need to add logging and/or telemetry.
```

Comment:
Nit: I see you emitting telemetry here is this comment still relevant? 

Not a blocker, can be removed later.

Reply:
Ah, good point, I can remove this, telemetry has been added.